src/main/java/suv/Helpers/Parser.java

```diff
@@ -0,0 +1,47 @@
+package suv.Helpers;
+
+import suv.Command.*;
+import suv.Task.FileStorage;
+
+/**
+ * The Parser class is responsible for parsing user input and executing corresponding commands.

```

Comment:
This class is doing multiple things. Within the scope of iP it is OK. However, this violates a design principle you will learn soon! Avoid doing more than one thing in a class in the tP.